### PR TITLE
Order codes columns

### PR DESCRIPTION
**Pull request by ElviaBth**
🎩 What? Why?
Possibility to sort or order the columns.

📌 Related Issues
#18 

📷 Screenshots
image

♥️ Thank you!

### CODE REVIEW

File: app/views/decidim/anonymous_codes/admin/codes/index.html.erb [hunk 1]
```diff
@@ -20,10 +20,10 @@
         <table class="table-list">
           <thead>
             <tr>
-              <th><%= t(".token") %></th>
-              <th><%= t(".available") %></th>
-              <th><%= t(".used") %></th>
-              <th><%= t(".usage_count") %></th>
+              <th><%= sort_link(query, :token, t(".token")) %></th>
+              <th><%= sort_link(query, :available, t(".available")) %></th>
+              <th><%= sort_link(query, :used, t(".used")) %></th>
```
Comment (microstudi) [inline app/views/decidim/anonymous_codes/admin/codes/index.html.erb:25]: please verify these two cases, as `available` and `used` **are not** columns in the database.

Comment (ElviaBth) [reply app/views/decidim/anonymous_codes/admin/codes/index.html.erb:25]: Ok

File: app/controllers/decidim/anonymous_codes/admin/codes_controller.rb [hunk 2]
```diff
@@ -64,6 +63,12 @@ def export
 
         private
 
+        def base_query
+          query = tokens
+          query = query.ransack(params[:q]).result if params[:q].present?
+          query.order(:token)
```
Comment (microstudi) [inline app/controllers/decidim/anonymous_codes/admin/codes_controller.rb]:
```suggestion
          tokens
```
this should suffice!

Comment (ElviaBth) [reply app/controllers/decidim/anonymous_codes/admin/codes_controller.rb]: Done

File: spec/system/admin/token_codes_spec.rb [hunk 3]
```diff
@@ -69,4 +69,53 @@
...
+      visit decidim_admin_anonymous_codes.code_groups_path
+
+      within find("tr", text: existing_group.title["en"]) do
+        expect(page).to have_link("List")
+        click_link "List"
+      end
+
+      click_on "Available?"
+
+      expect(page.body.index(used_token.token)).to be > page.body.index(available_token.token)
```
Comment (microstudi) [inline spec/system/admin/token_codes_spec.rb:109]: clever way!

**Review by microstudi — APPROVED** [review]
Nicely done!

**Pull request merged**